packages/amplify-frontend-android/lib/frontend-config-creator.js

```diff
@@ -137,15 +140,19 @@ function getCurrentAWSConfig(context) {
   const { amplify } = context;
   const projectPath = context.exeInfo ? context.exeInfo.localEnvInfo.projectPath : amplify.getEnvInfo().projectPath;
   const projectConfig = context.exeInfo ? context.exeInfo.projectConfig[constants.Label] : amplify.getProjectConfig()[constants.Label];
-  const frontendConfig = projectConfig.config;
-  const srcDirPath = path.join(projectPath, frontendConfig.ResDir, 'raw');
-
-  const targetFilePath = path.join(srcDirPath, constants.awsConfigFilename);
   let awsConfig = {};
 
-  if (fs.existsSync(targetFilePath)) {
-    awsConfig = amplify.readJsonFile(targetFilePath);
+  if (projectConfig) {
```

Comment:
Is this new logic? Why do we need it?

Reply:
I need to create the front end files from the external stack right now there is no way to write an external file path